packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardTypes.ts

```diff
@@ -10,41 +12,64 @@ export type StatConfig = {
   label: string
 }
 
+export type SelectedFacetConfig = {
+  // the column that should be faceted
+  facetColumnName: string
+  // specifies which column should be used to populate the selected facet value
+  facetValueColumnName: string
+}
+
 export type ButtonToExplorePageConfig = {
   // button's label
   label: string
   // column name in the data (table, view...) driving the page where the button is located
   // which is used to populate the button's path
   sourcePathColumnName: string
-  // defining SQL for the data (table, view...) driving the Explore page
-  // which is used in generating the button's query string
-  exploreDataSql?: string
-  // column name of the selected facet in the data (table, view...) driving the Explore page
-  // which is used in generating the button's query string
-  exploreDataFacetColumnName?: string
   // column name in the data (table, view...) driving the page where the button is located
-  // which is used to to populate the value for the selected facet in the button's query string
-  sourceDataFacetValueColumnName?: string
+  // which is used to populate the defining SQL for the data (table, view...) driving the Explore page
+  sourceExploreDataSqlColumnName?: string
+  // selected facets used in generating the button's query string,
+  // where the selected facet value is looked up in the data (table, view...) driving the page where the button is located:
+  //  - facetColumnName: name of the facet column in the data driving the Explore page
+  //  - facetValueColumnName: column name in the data driving the page where
+  //    the button is located and is used to populate the selected facet value
+  selectedFacetConfigs?: SelectedFacetConfig[]
+  // selected facets used in generating the button's query string,
+  // where the selected facet value is static
+  //  - columnName: name of the facet column in the data driving the Explore page
+  //  - facetValue: the selected facet value
+  staticSelectedFacets?: SelectedFacet[]
 }
 
-export type ReleaseCardLargeConfig = {
+export type ReleaseMetadataConfig = {
+  // specifies which column should be used to populate the release entity id
+  releaseEntityColumnName: string
```

Comment:
Small property rename for clarification (assuming this should always be an ENTITYID column)
```suggestion
  // specifies which column should be used to populate the release entity id
  releaseEntityIdColumnName: string
```